src/ondisk.rs

```diff
@@ -10416,6 +10428,19 @@ make_token_accessors! {
         MemDisplayPmuTrainingResults(default 0, id 0xb8a6_3eba) | pub get MemPmuTrainingResultOutput : pub set MemPmuTrainingResultOutput,
         /// See UMC::CH::ThrottleCtrl[ForcePwrDownThrotEn].
         MemForcePowerDownThrottleEnableTurin(default 1, id 0x1084_9d6c) | pub get MemForcePowerDownThrottleEnable : pub set MemForcePowerDownThrottleEnable, // used to be bool.
+        /// Whether PM should manage throttling--and measure sensor on DIMM
+        MemThermalThrottleMode(default 0, id 0xbce9_0051) | pub get MemThermalThrottleMode : pub set MemThermalThrottleMode, // note: default unknown
+
+        /// 40...100; point where memory throttling starts; in °C
+        MemThermalThrottleStartInC(default 40, id 0x1449_3d4b) | pub get u8 : pub set u8, // note: default unknown
+        /// 1...50; how many °C below MemThermalThrottleStartInC until we stop throttling
+        MemThermalThrottleSchmittGapInC(default 5, id 0x2205_08e7) | pub get u8 : pub set u8, // note: default unknown
```

Comment:
Why Schmitt instead of Hysteresis?

Reply:
Hysteresis is a behavior of a system: if the system depends on its (much) earlier history.

But for compatibility, it's better if I use AMD naming--so using Hysteresis after all.